packages/supertape/lib/supertape.d.ts

```diff
@@ -18,17 +18,17 @@ type Operators = {
 };
 
 type Test = Operators & OperatorStub & {
-    equal: (result: unknown, expected: unknown, message?: string) => Result;
-    notEqual: (result: unknown, expected: unknown, message?: string) => Result;
-    deepEqual: (result: unknown, expected: unknown, message?: string) => Result;
-    notDeepEqual: (result: unknown, expected: unknown, message?: string) => Result;
+    equal: (actual: unknown, expected: unknown, message?: string) => Result;
+    notEqual: (actual: unknown, expected: unknown, message?: string) => Result;
+    deepEqual: (actual: unknown, expected: unknown, message?: string) => Result;
+    notDeepEqual: (actual: unknown, expected: unknown, message?: string) => Result;
```

Comment:
Better to keep `result`, I'm actually never used name `actual`, and always use `result`.

Reply:
Okay, I saw that you updated everything else as well. I'll go ahead and change that.

Reply:
Changed in https://github.com/tommy-mitchell/supertape/commit/6de7b72346e11c6917ae48c8ec7e8d63d3c339ef